roles/openshift_hosted/tasks/registry/secure.yml

```diff
@@ -25,7 +25,8 @@
     signer_serial: "{{ openshift_master_config_dir }}/ca.serial.txt"
     hostnames:
     - "{{ docker_registry_service_ip.results.clusterip }}"
-    - docker-registry.default.svc.cluster.local
+    - docker-registry.default.svc
```

Comment:
@sdodson, @mtnbikenc,

Should we update these to have the name portion variable-ized?

`"{{ openshift_hosted_registry_name }}.default.svc"